changedetection.py

```diff
@@ -12,13 +12,19 @@
 import os
 
 def sigterm_handler(sig, frame):
+    print("Catch - SIGTERM")
     raise SystemExit
+def sigint_handler(sig, frame):
+    print("Catch - CTRL+C")
+    # Trigger SystemExit. But SIGINT already sent to the parse_process.
+    return
```

Comment:
This is the slightly detailed SIGINT mechanism. 
https://stackoverflow.com/a/31907519/20307768
https://unix.stackexchange.com/a/403605

So when you sent SIGINT, root changedetection.py and changedetectionio/changedetection.py are received at the same time.

You can check with `time.sleep(5)`. it is quite fun.

Reply:
I added this explanation as a commit message.